DuplicateElements.java

```diff
@@ -0,0 +1,27 @@
+
+// Java Program to print the duplicate elements of an array.
+
+import java.util.Scanner;
+
+public class DuplicateElements {
+    public static void main(String[] args) {
+        String[] myArray = new String[10];
+        Scanner scan = new Scanner(System.in);
+        System.out.println("Enter 10 elements of Array");
+        for (int i = 0; i < 10; i++) {
+            myArray[i] = scan.nextLine();
+        }
+        duplicates(myArray);
```

Comment:
try to be specific about output like (extractDublicate() / show*) while writing method name.